common/test/CMakeLists.txt

```diff
@@ -33,7 +45,7 @@ target_include_directories(common_test
 
 target_sources(common_test
   INTERFACE
-    ${CMAKE_CURRENT_SOURCE_DIR}/catch.hpp  
+    #${CMAKE_CURRENT_SOURCE_DIR}/catch.hpp  
```

Comment:
Seems like you should get remove this instead of commenting it out

Reply:
Thanks, removed